package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "auth-be",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --detectOpenHandles",
```

```suggestion
    "start": "node server.js",
    "start:watch": "nodemon server.js",
    "test": "jest --verbose --runInBand",
    "test:watch": "npm run test -- --watch",
    "lint": "eslint .",
    "pretest": "npm run lint"
```
Recommended scripts.